lib/components/Editor/CustomExtensions/Image/ExtensionConfig.js

```diff
@@ -130,10 +133,15 @@ const ImageExtension = Node.create({
 });
 
 const upload = async (file, url) => {
-  const uploader = new DirectUpload({ file, url });
-  const { blob_url: imageUrl } = await uploader.create();
+  if (file.size <= MAX_IMAGE_SIZE) {
+    const uploader = new DirectUpload({ file, url });
+    const { blob_url: imageUrl } = await uploader.create();
+
+    return imageUrl;
+  }
+  Toastr.error(`Image size should be less than ${MAX_IMAGE_SIZE / 1024} KB`);
```

Comment:
```suggestion

  const imageSizeInMB = MAX_IMAGE_SIZE / (1024 * 1024);
  Toastr.error(`Image size should be less than ${imageSizeInMB} KB`);
```

Showing the error message in MB is more readable and better in terms of UX.